gulpfile.js

```diff
@@ -21,7 +22,29 @@ gulp.task('tests', function () {
   });
 });
 
-gulp.task('javascript', function() {
+gulp.task('bundle-dist', function() {
+  var bundler = browserify({
+    entries: ['./index.js'],
+    debug: false,
+    standAlone: 'ServiceWorkerWare'
+  });
+
+  var bundle = function() {
+    return bundler
+      .bundle()
+      .pipe(source(getBundleName() + '.js'))
+      .pipe(buffer())
+      .pipe(removeLines({'filters': [
+        /performance/,
```

Comment:
Please, specify `.mark()` and `.measure()` explicitely to avoid conflicts with identifiers.